src/hotspot/share/prims/jvmti.xml

```diff
@@ -15456,7 +15456,7 @@ typedef void (JNICALL *jvmtiEventVMInit)
       Minor update to deprecate Heap functions 1.0.
   </change>
   <change date="27 April 2022" version="19.0.0">
-      Support for virtual threads:
+      Preview feature - Support for virtual threads:
```

Comment:
Maybe "Support for virtual threads as preview feature" or "Support for Virtual Threads (Preview)".

Reply:
Thanks. Fixed now.